Analyzers.props

```diff
@@ -1,7 +1,6 @@
 <Project>
   <ItemGroup>
     <PackageReference Include="DotNetAnalyzers.DocumentationAnalyzers" Version="1.0.0-beta.59" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.0" PrivateAssets="all" />
```

Comment:
Why is this removed?

Reply:
It is part of the SDK now. Build throws an error since the version in SDK is higher than this.

Reply:
That's good to know. Thanks!